crates/swift-bridge-ir/src/bridged_type.rs

```diff
@@ -32,6 +32,17 @@ mod built_in_tuple;
 mod shared_enum;
 pub(crate) mod shared_struct;
 
+/// Used to declare structures in a C header file.
+/// For example, struct FooStruct {struct BarStruct bar_struct;} is generated in the order of
+/// struct BarStruct {int32_t value;};
+/// struct struct FooStruct {struct BarStruct bar_struct;};
+///
+/// Please see https://github.com/chinedufn/swift-bridge/pull/225.
```

Comment:
I think that we can move this example to `CFFiStructDeclarationBookkeeping`

The docs on `CFFiStructDeclarationBookkeeping` should

1. Explain the problem
2. Explain how we use the struct to bookkeeping struct solve the problem

I think we should delete the link to the pull request. Anything relevant should be explained in the code doc.